configs/opencv-ovms/scripts/run_classification.sh

```diff
@@ -26,7 +26,8 @@ CONTAINER_NAME=classification"$cid_count"
 
 DOCKER_ENTRY=./classification/python/entrypoint.sh
 
-docker run --network host --env-file <(env) --privileged $rmDocker \
+docker run --network host --env-file "$PWD"/envs/classification.env --privileged $rmDocker \
```

Comment:
the env file breaks the current environment override capability from the command line... i think we still somehow writes out the key-value pairs into a temp file (maybe in /tmp/ folder) and then use that env file

Reply:
what command did you use that caused the break?

Reply:
This command works fine: **PIPELINE_PROFILE="object_detection" RENDER_MODE=1 sudo -E ./run.sh --workload ovms --platform dgpu.0 --inputsrc rtsp://127.0.0.1:8554/camera_0**

Reply:
@antoniomtz That is correct, From my understanding the order or priority is
1. -E ENV values set by the user when calling the script
2. ENV loaded from the file
3. ENV set in the local container

Reply:
> what command did you use that caused the break?

@antoniomtz  so the issue on the availability of environment overrides from the command line:
Take this classification as an example:
if you want to put different value for the same environment variable, say `CLASSIFICATION_OUTPUT_RESOLUTION` from the command line, i should be able to do it like this:
```console
PIPELINE_PROFILE="classification" CLASSIFICATION_OUTPUT_RESOLUTION="1080x640" sudo -E ./run.sh --workload ovms --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0
```
the environment variable CLASSIFICATION_OUTPUT_RESOLUTION inside the container should be `1080x640` that i put it in the command line; instead of the default value from the env file `classification.env` (1280x720).   

Brian and me discussed a little bit and we are ok to merge PR for your demo purpose for now and we will fix it in the generic docker script for this environment override issue.